manual.md

```
@@ -0,0 +1,163 @@
+### Build for Production
+
+Generate the static site into `_site/`:
+
+```
+JEKYLL_ENV=production bundle exec jekyll build
+```
+
+### Creating Content
+
+**New page:**
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)